src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -208,10 +209,16 @@ const OPERATOR_MAP: {[op:string]: string} = {
     ">": ">"
 };
 
-export function getClinicalAttributeOverlay(displayName: string, description: string): JSX.Element {
-    return <div style={{maxWidth: '500px'}}>
-        <b>{displayName}</b><br/>
-        {description}
+export function getClinicalAttributeOverlay(displayName: string, description: string, clinicalAttributeId?: string): JSX.Element {
+    const comparisonDisplayName = displayName.toLowerCase().trim();
+    let comparisonDescription = description.toLowerCase().trim();
+    if (comparisonDescription.substring(comparisonDescription.length - 1) === '.') {
+        comparisonDescription = comparisonDescription.substring(0, comparisonDescription.length - 1);
+    }
+    return <div style={{maxWidth: '500px', display: 'flex', flexDirection: 'column'}}>
```

Comment:
i'm not sure you need this flex stuff.  where you want it to break line, just use div and instead of span